app/main.py

```diff
@@ -2,0 +1,18 @@
+from typing import List
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: List[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+
+    if masks_to_buy:
+        return f"Friends should buy {masks_to_buy} masks"
```

Comment:
Again, the error message could be more descriptive. You could include the names of the friends who need to buy masks in the message. Something like this: `return f'{friends_who_need_masks} should buy masks'`.